packages/ui/react-ui-table/src/components/Toolbar/Toolbar.tsx

```diff
@@ -46,30 +45,45 @@ const Editing = () => {
   const { t } = useTranslation(translationKey);
 
   return (
-    <>
-      <ToolbarButton
-        value='add-row'
-        icon='ph--plus--regular'
-        data-testid='table.toolbar.add-row'
-        onClick={() => onAction?.({ type: 'add-row' })}
-      >
-        {t('add row')}
-      </ToolbarButton>
-    </>
+    <ToolbarButton
+      value='add-row'
+      icon='ph--plus--regular'
+      data-testid='table.toolbar.add-row'
+      onClick={() => onAction?.({ type: 'add-row' })}
+    >
+      {t('add row')}
+    </ToolbarButton>
   );
 };
 
-const Actions = () => {
+const Actions = ({ viewDirty }: { viewDirty?: boolean }) => {
   const { onAction } = useToolbarContext('Actions');
   const { t } = useTranslation(translationKey);
 
   return (
     <>
+      {viewDirty && (
+        <IconButton
+          value='save-view'
+          icon='ph--floppy-disk--regular'
+          data-testid='table.toolbar.save-view'
+          onClick={() => {
+            if (onAction) {
+              onAction({ type: 'save-view' });
+            }
+          }}
+          label={t('save view')}
```

Comment:
```suggestion
          label={t('save view label')}
```
I think this is generally the pattern @thure has used for translation keys elsewhere